dask_cuda/disk_io.py

```diff
@@ -164,11 +164,16 @@ def disk_write(path: str, frames: Iterable, shared_filesystem: bool, gds=False)
     cuda_frames = tuple(hasattr(f, "__cuda_array_interface__") for f in frames)
     frame_lengths = tuple(map(nbytes, frames))
     if gds and any(cuda_frames):
-        import cucim.clara.filesystem as cucim_fs
+        import kvikio
 
-        with cucim_fs.open(path, "w") as f:
+        futs = []
+        with kvikio.CuFile(path, "w") as f:
             for frame, length in zip(frames, frame_lengths):
-                f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0)
+                futs.append(
+                    f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0)
```

Comment:
If `buf` is on host (as some frames like the header could be), this will raise an error. Would be good to make this more robust to that kind of usage. Raised issue ( https://github.com/rapidsai/kvikio/issues/76 )